test/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/UKAddressLookupServiceImplSpec.scala

```diff
@@ -119,6 +119,120 @@ class UKAddressLookupServiceImplSpec extends WordSpec with Matchers with MockFac
           ).isLeft shouldBe true
         }
 
+        "address line1 is more than 35 characters" in {
+          val json = Json.parse(
+            """
+              |[
+              |  {
+              |    "address": {
+              |      "lines": [ "address line1 is more than 35 characters" ],
+              |      "town": "town",
+              |      "county" : "county",
+              |      "postcode": "ZZ1Z 4AB"
+              |    }
+              |  }
+              |]
+              |""".stripMargin
+          )
+
+          mockLookupAddress(postcode)(Right(HttpResponse(200, Some(json))))
+
+          await(
+            service.lookupAddress(postcode, None).value
+          ).exists(_.addresses.isEmpty) shouldBe true
```

Comment:
testing like this would result in error messages that are more helpful if something goes wrong with the test:
```
await(service.lookupAddress(postcode, None).value) shouldBe Right(AddressLokupResult(...,..., List.empty))
```
Otherwise, we'd see `false was not equal to true` which as helpful as seeing the whole result itself